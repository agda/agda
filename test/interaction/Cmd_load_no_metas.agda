-- Andreas, 2024-08-02, cover command Cmd_load_no_metas

f = ?

-- This should fail as no unsolved metas are allowed.
