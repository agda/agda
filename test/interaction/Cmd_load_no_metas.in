top_command (cmd_load_no_metas currentFile)