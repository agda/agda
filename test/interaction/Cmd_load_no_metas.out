(agda2-status-action "")
(agda2-info-action "*Type-checking*" "" nil)
(agda2-highlight-clear)
(agda2-info-action "*Error*" "error: [UnsolvedMetaVariables] Unsolved metas at the following locations: Cmd_load_no_metas.agda:3,1-2 error: [UnsolvedInteractionMetas] Unsolved interaction metas at the following locations: Cmd_load_no_metas.agda:3,5-6" nil)
(agda2-highlight-load-and-delete-action)
(agda2-status-action "")
